pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -326,7 +346,7 @@ sub _upgrade_avplus_text {
 
 sub _install_av_text {
     my ($self) = @_;
-    my $install_av_url = $self->base_path('scripts13/install_imunifyav_SECURITYADVISOR');
+    my $install_av_url = $self->base_path('scripts14/install_imunifyav_SECURITYADVISOR');
```

Comment:
should not this use `_get_script_number` to use the accurate script depending on cPanel&WHM version?

Reply:
For IAV and IAVP there is only one script# to use.

_get_script_number is for Imunify360 only

Reply:
perfect